lib/puppet/pops/types/types.rb

```diff
@@ -403,7 +451,7 @@ def eql?(o)
     self.class == o.class && @type == o.type
   end
 
-  def simple_name
+  def self.simple_name
     # since this the class is inconsistently named PType and not PTypeType
```

Comment:
Sure. While we're at it, we should also change `PHostClassType` to `PClassType` since type is named `Class`. Separate JIRA?